src/pyOpenMS/pxds/MetaboTargetedAssay.pxd

```diff
@@ -0,0 +1,35 @@
+from Types cimport *
+from BaseFeature cimport *
+from MSExperiment cimport *
+from MSSpectrum cimport*
+from SiriusMSFile cimport *
+from libcpp.vector cimport vector as libcpp_vector
+from libcpp.map cimport map as libcpp_map
+from libcpp.pair cimport pair as libcpp_pair
+
+
+
+cdef extern from "<OpenMS/ANALYSIS/TARGETED/MetaboTargetedAssay.h>" namespace "OpenMS":
+
+    cdef cppclass MetaboTargetedAssay "OpenMS::MetaboTargetedAssay":
+
+       MetaboTargetedAssay() nogil except +
+       MetaboTargetedAssay(MetaboTargetedAssay) nogil except + #wrap-ignore
+
+
+
+       libcpp_vector[MetaboTargetedAssay] extractMetaboTargetedAssay(MSExperiment spectra,
+                                                                     libcpp_map[ BaseFeature,libcpp_vector[size_t] ] feature_ms2_spectra_map,
```

Comment:
> libcpp_map[ BaseFeature,libcpp_vector[size_t] ]

that probably doesnt work, you may have to remove the wrapper here

Reply:
tried to set it to FeatureMapping::FeatureToMs2Indices feature_ms2_index, but error persits. 
so it is probably that one: 
libcpp_vector[ libcpp_pair[ SiriusMSFile_CompoundInfo,MSSpectrum ] ]& v_cmp_spec